tools/tck-api/src/test/scala/org/opencypher/tools/tck/values/CypherValueTest.scala

```diff
@@ -153,6 +151,28 @@ class CypherValueTest extends AnyFunSuite with Matchers {
       not equal(CypherValue("[[(:D:A), (:C:B)], [(:DD:AA), (:CC:BB)]]", orderedLists = true))
   }
 
+  test("nodes with properties") {
+    assertReallyEqual(
+      CypherValue("[(:A {a: 1, b: 2}), (:A {b: 4, a: 3})]", orderedLists = true),
+      CypherValue("[(:A {b: 2, a: 1}), (:A {a: 3, b: 4})]", orderedLists = true),
+    )
+    assertReallyEqual(
+      CypherValue("[(:A {a: 1, b: 2}), (:A {b: 4, a: 3})]", orderedLists = false),
+      CypherValue("[(:A {a: 3, b: 4}), (:A {b: 2, a: 1})]", orderedLists = false),
+    )
+  }
+
+  test("relationships with properties") {
+    assertReallyEqual(
+      CypherValue("[[:A {a: 1, b: 2}], [:A {b: 4, a: 3}]]", orderedLists = true),
+      CypherValue("[[:A {b: 2, a: 1}], [:A {a: 3, b: 4}]]", orderedLists = true),
+    )
+    assertReallyEqual(
+      CypherValue("[[:A {a: 1, b: 2}], [:A {b: 4, a: 3}]]", orderedLists = false),
+      CypherValue("[[:A {a: 3, b: 4}], [:A {b: 2, a: 1}]]", orderedLists = false),
+    )
+  }
+
   private def assertReallyEqual(a: CypherValue, b: CypherValue): Unit = {
     a shouldBe a
     b shouldBe b
```

Comment:
We have that below. But since the equals methods are so complex for list values I wanted to test this too. It's easy to implement equals that will return false even if we're comparing the same instance, so it provides some value.